mabl snippets/businessDayAdd.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Return a date in mm/dd/yyyy format offset by a user specified number of business days (parameterized)
+ *
+ * @param {object} mablInputs - Object containing mabl inputs such as variables (mablInputs.variables).
+ *                              Use mablInputs.variables.user for user defined variables
+ *                              (For example myVar may be accessed as mablInputs.variables.user.myVar)
+ *
+ * @param {function} callback - A callback function that must be called to complete
+ *                              the javascript step and provide a value to the following
+ *                              steps of the flow/journey. A return statement from this
+ *                              function call will not provide any results for use
+ *                              in the following steps in this flow or journey.
+ */
+ function mablJavaScriptStep(mablInputs, callback, daysToAdd = undefined) {
+    let today = new Date();
+
+   if(daysToAdd > 0){
+        //Adds days for each business day
+        for (let i = 1; i <= daysToAdd; i++) {
```

Comment:
Should we be using `var` here for ES5 compatibility?

Reply:
Resolved :)